### PR TITLE
feat: [MOD-728] DecideID_Assets as custom canister

### PR DESCRIPTION
**Pull request by VitalyGitDev**
---
### Summary
DecideID asset canister refactored to be custom canister with Rust code serving http_requests and certified assets.

### Issues
[*Link to the issue](https://linear.app/modclub/issue/MOD-728/refactor-decideid-asset-canister-to-be-custom-asset-caniser)

### Type of Change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Documentation update

### Checklist
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published.

### Test
-
### Additional Context
-

### CODE REVIEW

File: src/decideid_assets/app/actors_by_env.ts
Comment (xwdeng) [inline src/decideid_assets/app/actors_by_env.ts:1]: Should not have this file in the commit. ( This is also configured in .gitignore )

File: src/decideid_assets/app/canister_types.ts
Comment (xwdeng) [inline src/decideid_assets/app/canister_types.ts:1]: Should not have this file in the commit. ( This is also configured in .gitignore )

File: src/decideid_assets/app/components/header/Header.tsx [hunk 1]
```diff
@@ -1,15 +1,13 @@
-import React, { useContext } from 'react';
+import React, { useContext } from "react";
```
Comment (xwdeng) [inline src/decideid_assets/app/components/header/Header.tsx:1]: you have a different setup locally that will change the format of the tsx file ?

File: src/decideid_assets/decideid_assets.did [hunk 2]
```diff
@@ -0,0 +1,102 @@
+/// Specification of a requested credential.
```
[the remark this answers is not in the document]
Comment (VitalyGitDev) [reply src/decideid_assets/decideid_assets.did:1]: At the moment - yes, because we have "custom" config for the canister in dfx.json

**Pull request merged**